lisa/node.py

```diff
@@ -191,6 +198,29 @@ def support_sudo(self) -> bool:
     def is_connected(self) -> bool:
         return self._shell is not None and self._shell.is_connected
 
+    @property
+    def local_log_path(self) -> Path:
+        if not self._local_log_path:
```

Comment:
Why not return early on the good case and avoid the indentation?

Reply:
In some coding guidline, it recommends to single exit entry in each function. I'm struggling sometime, but following it.